### PR TITLE
Add KCC 2021 contest article

### PR DESCRIPTION
**Pull request by Garalulu**
Added Korean Charting Contest 2021 & KCC Index Page. (EN / KR)

### CODE REVIEW

File: wiki/Contests/KCC/1/en.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+tags:
+  - KCC2021
+  - KCC 2021
+---
+
+# KCC 2021
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:7]:
```suggestion
# Korean Charting Contest 2021
```

File: wiki/Contests/KCC/1/en.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+![KCC2021 Logo](img/banner.png)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:9]:
```suggestion
![KCC2021 banner](img/banner.png)
```

File: wiki/Contests/KCC/1/en.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The **Korean Charting Contest 2021** (***KCC2021***) is a osu!mania mapping contest for Koreans.  This edition was the first installment of the KCC.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:11]:
```suggestion
The **Korean Charting Contest 2021** (***KCC2021***) is a osu!mania mapping contest for users from ![][flag_KR] South Korea.  It was the first installment of the Korean Charting Contest.
```

File: wiki/Contests/KCC/1/en.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+## Contest schedule
+
+| Event | Timestamp (UTC +9) |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:15]:
```suggestion
| Event | Timestamp (UTC+09) |
```

File: wiki/Contests/KCC/1/en.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+| --: | :-- |
+| Registration Phase, Songs Voting | 2021-02-01/2021-02-07 |
+| Submitting Phase | 2021-02-08/2021-03-07 |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:18]:
```suggestion
| Submission phase | 2021-02-08/2021-03-07 |
```

File: wiki/Contests/KCC/1/en.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+![KCC2021 Logo](img/banner.png)
+
+The **Korean Charting Contest 2021** (***KCC2021***) is a osu!mania mapping contest for Koreans.  This edition was the first installment of the KCC.
+
+## Contest schedule
+
+| Event | Timestamp (UTC +9) |
+| --: | :-- |
+| Registration Phase, Songs Voting | 2021-02-01/2021-02-07 |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:17]:
```suggestion
| Registration phase, song voting | 2021-02-01/2021-02-07 |
```

File: wiki/Contests/KCC/1/en.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+| Submitting Phase | 2021-02-08/2021-03-07 |
+| Judging Phase | 2021-03-08/2021-03-21 |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:19]:
```suggestion
| Judging phase | 2021-03-08/2021-03-21 |
```

File: wiki/Contests/KCC/1/en.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+| Results Announcement | 2021-03-22 |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:20]:
```suggestion
| Results announcement | 2021-03-22 |
```

File: wiki/Contests/KCC/1/en.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!mania mode only.
+- Only Koreans can participate in this contest. If your profile flag is not showing that you're in Korea because of some reasons like foreign residence, you can still participate as long as you're able to verify that you are Korean.
+- You should pick a song that you want to map when you register. one song for Instrumental / Vocal each. You can map one of the two songs selected after the vote ends.
+- Submission must be loadable and readable by osu! client.
+- Submission must be in ``.osz`` format.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:30]:
```suggestion
- Submission must be in `.osz` format.
```

File: wiki/Contests/KCC/1/en.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+- Alice Schach and the Magic Orchestra - Chocolate Missile (2:57) / 175bpm
+- HyuN - Last Paradise (feat. Serentium) (2:42) / 89bpm
+- Memme - Geurida (2:45) / 180bpm
+- mk feat.橘花音 - Awaking Beat-From the next generation- (3:51) / 178bpm
+- **YUC'e - Sengoku HOP (3:25) / 110bpm**
+
+### Vote results
+
+![KCC2021 Vote Results](img/voteresult.jpg)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:63]:
```suggestion
![KCC2021 voting results](img/voteresult.jpg)
```

File: wiki/Contests/KCC/1/en.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+## Judging Criteria
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:65]:
```suggestion
## Judging criteria
```
should be a part of `Ruleset`

File: wiki/Contests/KCC/1/en.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+- **Creativity (25 pts):** How well does the submission show originality and uniqueness compared with the other submissions with reasonable respect to the playability?
+- **Judge's Impression (25 pts):** This is the part of the score left to the Judges’ personal preferences on the submission
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | 40,000 KRW, profile badge |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 20,000 KRW |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 10,000 KRW |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:80]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 40,000 KRW, profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20,000 KRW |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10,000 KRW |
```

File: wiki/Contests/KCC/1/en.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+![KCC2021 Badge](img/badge.png)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:82]:
```suggestion
![KCC2021 badge](img/badge.png)
```

File: wiki/Contests/KCC/1/en.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Organizer | ![][flag_KR] [Garalulu](https://osu.ppy.sh/users/757783) |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:88]:
```suggestion
| Organiser | ![][flag_KR] [Garalulu](https://osu.ppy.sh/users/757783) |
```

File: wiki/Contests/KCC/1/en.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+| Place | Mapper |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_KR] [11Bit](https://osu.ppy.sh/users/14804526) |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_KR] [Sherie](https://osu.ppy.sh/users/9113475) |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_CA] [arpia97](https://osu.ppy.sh/users/6363008) |
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1230057)
+- [Results Sheet](https://docs.google.com/spreadsheets/d/1O0Ygpning0te62S850M42oPo0lCYd1Ct8VeYnAWRcYE/edit?usp=sharing)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:104]:
```suggestion
- [Results sheet](https://docs.google.com/spreadsheets/d/1O0Ygpning0te62S850M42oPo0lCYd1Ct8VeYnAWRcYE/edit?usp=sharing)
```

File: wiki/Contests/KCC/1/en.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+- [All Entries](https://lulu.s-ul.eu/D2M3x9LH)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:105]:
```suggestion
- [Entries](https://lulu.s-ul.eu/D2M3x9LH)
```

**Review by TPGPL — CHANGES_REQUESTED** [review]
`kr.md` files should be  renamed to `ko.md`

File: wiki/Contests/KCC/1/en.md [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+| Submission phase | 2021-02-08/2021-03-07 |
+| Judging phase | 2021-03-08/2021-03-21 |
+| Results announcement | 2021-03-22 |
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!mania mode only.
+- Only Koreans can participate in this contest. If your profile flag is not showing that you're in Korea because of some reasons like foreign residence, you can still participate as long as you're able to verify that you are Korean.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:27]:
```suggestion
- Only Koreans can participate in this contest. If your profile flag is not showing that you're in ![][flag_KR] South Korea for reasons like foreign residence, you can still participate as long as you're able to verify that you are Korean.
```

File: wiki/Contests/KCC/1/en.md [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+- You should pick a song that you want to map when you register. one song for Instrumental / Vocal each. You can map one of the two songs selected after the vote ends.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:28]:
```suggestion
- You should pick a song that you want to map when you register (one song for Instrumental / Vocal each). You can map one of those two songs after the voting ends.
```

File: wiki/Contests/KCC/1/en.md [hunk 19]
```diff
@@ -0,0 +1,110 @@
...
+- Submission must be loadable and readable by osu! client.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:29]:
```suggestion
- The osu! client must be able to load and read your submission.
```

File: wiki/Contests/KCC/1/en.md [hunk 20]
```diff
@@ -0,0 +1,110 @@
...
+- Submission must be in `.osz` format.
+- Custom hitsounds are allowed.
+- Title, Artist, MP3, Source, Tags and Background must NOT be modified. However, preview point can be customized.
+- No video, skin and storyboard are allowed. However, storyboard hitsounds are allowed.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:33]:
```suggestion
- Videos, skins and storyboards are NOT allowed. However, storyboard hitsounds can be modified.
```

File: wiki/Contests/KCC/1/en.md [hunk 21]
```diff
@@ -0,0 +1,110 @@
...
+- Submission must be 4K ~ 9K.
+- Submission must be full comboed (1,000,000 score) by Auto mod.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:35]:
```suggestion
- The Auto mod must be able to full combo (1,000,000 score) your submission.
```

File: wiki/Contests/KCC/1/en.md [hunk 22]
```diff
@@ -0,0 +1,110 @@
...
+- Submission must be finished by the contestant only. No collaboration is allowed.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:36]:
```suggestion
- Submission must be finished by the contestant alone. No collaboration is allowed.
```

File: wiki/Contests/KCC/1/en.md [hunk 23]
```diff
@@ -0,0 +1,110 @@
...
+- Submission must not be uploaded to the osu! website through the Beatmap Submission System (BSS). All contest submissions are anonymous during the judging phases to avoid bias. 
+- One and only one difficulty is allowed. If multiple difficulties are submitted, only the one with the highest star difficulty will be accepted.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:38]:
```suggestion
- Only one beatmap difficulty is allowed. If multiple difficulties are submitted, then the one with the highest star difficulty will be accepted.
```

File: wiki/Contests/KCC/1/en.md [hunk 24]
```diff
@@ -0,0 +1,110 @@
...
+- Only the following difficulty are allowed: Insane or Higher. Please check the Difficulty-specific Rule in the [ranking criteria](/wiki/Ranking_Criteria).
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:39]:
```suggestion
- Only the following difficulties are allowed: Insane or higher. Please check the difficulty-specific rules in the [ranking criteria](/wiki/Ranking_Criteria).
```

File: wiki/Contests/KCC/1/en.md [hunk 25]
```diff
@@ -0,0 +1,110 @@
...
+- If you submit several maps, the last one would be your final entry.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:40]:
```suggestion
- If you submit several maps, the last one will be your final entry.
```

File: wiki/Contests/KCC/1/en.md [hunk 26]
```diff
@@ -0,0 +1,110 @@
...
+- Submitting entries means understanding and agreeing the ruleset above.
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:41]:
```suggestion
- Submitting entries means understanding and agreement to the ruleset above.
```

File: wiki/Contests/KCC/1/en.md [hunk 27]
```diff
@@ -0,0 +1,110 @@
...
+
+![KCC2021 voting results](img/voteresult.jpg)
+
+## Judging criteria
+
+For each eligible submitted difficulty, the juding panel will award points based on the following criteria:
+
+- **Expertise (25 pts):** How well does the submission demonstrate techniques regarding structure, music representation, flow, hitsounding, etc?
+- **Cohesion (25 pts):** How consistent is the submission regarding structure, music representation, flow, hitsounding etc especially between different sections?
+- **Creativity (25 pts):** How well does the submission show originality and uniqueness compared with the other submissions with reasonable respect to the playability?
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/en.md:71]:
```suggestion
- **Expertise (25 pts):** How well does the submission demonstrate techniques regarding structure, music representation, flow, hitsounding, etc.
- **Cohesion (25 pts):** How consistent is the submission regarding structure, music representation, flow, hitsounding etc especially between different sections.
- **Creativity (25 pts):** How well does the submission show originality and uniqueness compared with the other submissions with reasonable respect to the playability.
```

File: wiki/Contests/KCC/1/ko.md [hunk 28]
```diff
@@ -0,0 +1,110 @@
...
+- **Creativity (25 pts):** 다른 맵과 비교했을 때 플레이성 면에서 독창성과 특별함을 가지고 있는가?
+- **Judge's Impression (25 pts):** 심사위원들의 개인적 선호도를 나타내는 점수
+
+## 상품
+
+| 순위 | 상품 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1위") | 40,000 KRW, 프로필 뱃지 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2위") | 20,000 KRW |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3위") | 10,000 KRW |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/ko.md:80]: text in square brackets can be translated

File: wiki/Contests/KCC/1/ko.md [hunk 29]
```diff
@@ -0,0 +1,110 @@
...
+| 디자이너 | ![][flag_KR] [POCARI SWEAT](https://osu.ppy.sh/users/5082685) |
+| 심사위원 | ![][flag_PL] [_underjoy](https://osu.ppy.sh/users/6392061), ![][flag_KR] [Garalulu](https://osu.ppy.sh/users/757783), ![][flag_GT] [Hoto Cocoa](https://osu.ppy.sh/users/6974536), ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754), ![][flag_KR] [Pengdoll](https://osu.ppy.sh/users/6392061) |
+
+## 결과
+
+| 순위 | 매퍼 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1위") | ![][flag_KR] [11Bit](https://osu.ppy.sh/users/14804526) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2위") | ![][flag_KR] [Sherie](https://osu.ppy.sh/users/9113475) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3위") | ![][flag_CA] [arpia97](https://osu.ppy.sh/users/6363008) |
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/ko.md:99]: text in square brackets can be translated

File: wiki/Contests/KCC/1/ko.md [hunk 30]
```diff
@@ -0,0 +1,110 @@
...
+
+- Alice Schach and the Magic Orchestra - Chocolate Missile (2:57) / 175bpm
+- HyuN - Last Paradise (feat. Serentium) (2:42) / 89bpm
+- Memme - Geurida (2:45) / 180bpm
+- mk feat.橘花音 - Awaking Beat-From the next generation- (3:51) / 178bpm
+- **YUC'e - Sengoku HOP (3:25) / 110bpm**
+
+### 투표 결과
+
+![KCC2021 voting results](img/voteresult.jpg)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/ko.md:63]: text in square brackets can be translated

File: wiki/Contests/KCC/1/ko.md [hunk 31]
```diff
@@ -0,0 +1,110 @@
+---
+tags:
+  - KCC2021
+  - KCC 2021
+---
+
+# Korean Charting Contest 2021
+
+![KCC2021 banner](img/banner.png)
```
Comment (TPGPL) [inline wiki/Contests/KCC/1/ko.md:9]: text in square brackets can be translated

File: wiki/Contests/KCC/1/en.md [hunk 32]
```diff
@@ -0,0 +1,110 @@
...
+- mk feat.橘花音 - Awaking Beat-From the next generation- (3:51) / 178bpm
+- **YUC'e - Sengoku HOP (3:25) / 110bpm**
+
+### Voting results
+
+![KCC2021 voting results](img/voteresult.jpg)
+
+## Judging criteria
+
+For each eligible submitted difficulty, the juding panel will award points based on the following criteria:
```
Comment (Walavouchey) [inline wiki/Contests/KCC/1/en.md:67]:
```suggestion
For each eligible submitted difficulty, the judging panel will award points based on the following criteria:
```

File: wiki/Contests/KCC/1/en.md [hunk 33]
```diff
@@ -0,0 +1,110 @@
...
+
+- **Expertise (25 pts):** How well does the submission demonstrate techniques regarding structure, music representation, flow, hitsounding, etc.
+- **Cohesion (25 pts):** How consistent is the submission regarding structure, music representation, flow, hitsounding etc especially between different sections.
+- **Creativity (25 pts):** How well does the submission show originality and uniqueness compared with the other submissions with reasonable respect to the playability.
+- **Judge's Impression (25 pts):** This is the part of the score left to the Judges’ personal preferences on the submission.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/1/en.md:72]: These are formed as questions, but don't have question marks, although I think they're more appropriate as statements.

```suggestion
- **Expertise (25 pts):** How well the submission demonstrates techniques regarding structure, music representation, flow, hitsounding, etc.
- **Cohesion (25 pts):** How consistent the submission is regarding structure, music representation, flow, hitsounding etc especially between different sections.
- **Creativity (25 pts):** How well the submission shows originality and uniqueness compared with the other submissions with reasonable respect to the playability.
- **Judge's Impression (25 pts):** This is the part of the score left to the Judges' personal preferences on the submission.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
Otherwise alright for the English part 👍

**Review by TPGPL — APPROVED** [review]

**Pull request merged**